inc/Api/Api.php

```diff
@@ -166,123 +166,150 @@ public function register_api_information() {
-				$sanitized_user                  = [];
-				$sanitized_cart_autoresponder    = [];
-				$sanitized_syncronize_additional = [];
-				$sanitized_cart_options          = [];
-				if ( is_array( $user ) ) {
-					foreach ( $user as $key => $value ) {
-						$sanitized_user[ $key ] = wp_unslash( sanitize_text_field( $value ) );
-					}
-				}
-
-				if ( is_array( $cart_autoresponder ) ) {
-					foreach ( $cart_autoresponder as $key => $value ) {
-						$sanitized_cart_autoresponder [ $key ] = wp_unslash( sanitize_text_field( $value ) );
-					}
-				}
-
-				if ( isset( $autoresponders['syncronize_additional'] ) &&
-					is_array( $autoresponders['syncronize_additional'] ) ) {
-					foreach ( $autoresponders['syncronize_additional'] as $key => $value ) {
-						$sanitized_syncronize_additional[ $key ] = wp_unslash( sanitize_text_field( $value ) );
-					}
-				}
-
-				if ( isset( $autoresponders['cart_options'] ) &&
-					is_array( $autoresponders['cart_options'] ) ) {
-					foreach ( $autoresponders['cart_options'] as $key => $value) {
-						$sanitized_cart_options [ $key ] = wp_unslash( sanitize_text_field( $value ) );
-					}
-				}
-
-				// Sanitize Abandoned cart delay, cutoff time and enabled status.
-				$cart_cutoff_time      = (int) wp_unslash( sanitize_text_field( $autoresponders['cart_cutoff'] ) );
-				$cart_enabled          = isset( $autoresponders['enable_cart'] ) ? 1 : 0;
-				$enabled               = isset( $autoresponders['enable'] ) ? 1 : 0;
-				$syncronize_additional = ( $sanitized_syncronize_additional ) ? implode( ',', $sanitized_syncronize_additional ) : null;
-				$cart_options          = isset( $sanitized_cart_options ) ? implode( ',', $sanitized_cart_options ) : null;
-
-				// Save data to database.
-				global $wpdb;
-				// Smaily table name.
-				$table_name = $wpdb->prefix . 'smaily';
-
-				// Check if abandoned cart is enabled.
-				if ( $cart_enabled ) {
-					// Check if autoresponder for cart is selected.
-					if ( empty( $sanitized_cart_autoresponder ) ) {
-						// Return error if no autoresponder for abandoned cart.
-						echo wp_json_encode(
-							array(
-								'error' => esc_html__( 'Select autoresponder for abandoned cart!', 'smaily' ),
-							)
-						);
-						wp_die();
-					}
-					// Check if cart cutoff time is valid.
-					if ( $cart_cutoff_time < 10 ) {
-						echo wp_json_encode(
-							array(
-								'error' => esc_html__( 'Abandoned cart cutoff time value must be 10 or higher!', 'smaily' ),
-							)
-						);
-						wp_die();
-					}
-				}
-
-				// Update DB with user values if autoresponder selected.
-				$result = $wpdb->update(
-					$table_name,
+			echo wp_json_encode(
+				array(
+					'error' => esc_html__( 'Missing form data!', 'smaily' ),
+				)
+			);
+			wp_die();
+		}
+
+		if ( ! current_user_can( 'manage_options' ) ) {
+			echo wp_json_encode(
+				array(
+					'error' => esc_html__( 'You are not authenticated to edit settings!', 'smaily' ),
```

Comment:
"authorized" is a better word to use in this sentence instead of "authenticated".

Reply:
Fixed in https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/39/commits/20c8b71636360e81ee54db0622a777406774c1f9